app/controllers/appointments_controller.rb

```diff
@@ -38,6 +41,37 @@ def update
 
   private
 
+  def apply_search_filters
```

Comment:
Here is the query logic - eventually I'd like to have this in a `PatientSummarySearch` object or something, but want to know where else we are going to be searching patients like this w/ other dashboards.